examples/ssd1675_2.13_monochrome.py

```diff
@@ -0,0 +1,46 @@
+"""Simple test script for 2.13" 250x122 monochrome display.
+
+Supported products:
+  * Adafruit 2.13" Monochrome ePaper Display Breakout
+    * https://www.adafruit.com/product/4197
+  """
+
+import time
+import board
+import busio
+import displayio
+import adafruit_ssd1675
+
+displayio.release_displays()
+
+# This pinout works on a Metro and may need to be altered for other boards.
+
+# For breadboarding
+spi = busio.SPI(board.SCL, board.SDA)
```

Comment:
Let's use the board.SPI() for the SPI pins instead of I2C.

Reply:
yes please!

Reply:
Swapped to board.SPI, documented and tested using Feather M4 and breakout.